platform/src/main/resources/config/anchor-config-default-values.yaml

```diff
@@ -223,151 +199,120 @@ sep1:
   ## SEP-1 stellar.toml. See the SEP-10 section `Verifying Client Application Identity` for more information
   ## (https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0010.md#verifying-client-application-identity).
-
   #
   client_attribution_required: false
-
   ## @param: client_attribution_allow_list
   ## Set the white list of the client domain. The domains are comma-separated.
   #
   client_attribution_allow_list:
-
   ## @param: client_attribution_deny_list
   ## Set the black list of the client domain. The domains are comma-separated.
   #
   client_attribution_deny_list:
-
   ## @param: auth_timeout
   ## @type: integer
   ## Set the authentication challenge transaction timeout in seconds. An expired signed transaction will be rejected.
   ## This is the timeout period the client must finish the authentication process. (ie: sign and respond the challenge
   ## transaction).
   #
   auth_timeout: 900
-
   ## @param: jwt_timeout
   ## @type: integer
   ## Set the timeout in seconds of the authenticated JSON Web Token. An expired JWT will be rejected.
   ## This is the timeout period after the client has authenticated.
   #
   jwt_timeout: 86400
-
   ## @param: omnibus_account_list
   ## A comma-separated list of omnibus accounts (comma separated).
   ## The SEP-10 omnibus account is described at:
   ##     https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0010.md#memos
   #
   omnibus_account_list:
-
   ## @param: require_known_omnibus_account
   ## @type: bool
   ## Whether to require authenticating clients to be in the omnibusAccountList
   #
   require_known_omnibus_account: false
 
 sep12:
-
   ## @param: enabled
   ## @type: bool
   ## Whether to enable SEP-12 for this instance of AP server.
   #
   enabled: false
 
 sep24:
-  ## SEP24 in experimental.
-
   ## @param: enabled
   ## @type: bool
   ## Whether to enable SEP-24 for this instance of AP server.
   #
   enabled: false
-
-  ## @param: interactive_jwt_expiration
-  ## @type: integer
-  ## Set the timeout in seconds of the JSON Web Token returned with the embedded interactive url of the SEP-24 process.
-  ## An expired JWT will be rejected.
-  ##
-  ## If the interactive flow needs to access the platform server, the interactive process must finish within
-  ## the specified timeout period.
-  #
-  interactive_jwt_expiration: 3600
-
   ## @param: interactive_url
   # Configures the interactive URL where the platform server will redirect to start the SEP-24 interactive flow.
-  # If the type is `simple`, sep24.interactive_url.simple will be read to construct the url.
   #   base_uri: is the base_uri used to construct the interactive url
-  #   txn_fields: is the list of transaction fields that will be appended to the interactive url as query parameters
+  #   txn_fields: is an optional list of transaction fields that will be appended to the interactive url as query parameters. If empty, no fields from transaction will be appended.
```

Comment:
We'll make sure to add documentation about how these aren't necessarily the values stored in the AP, but can we add those comments here too?

Reply:
Hey @JakeUrban, we only search for the transaction fields that are stored in the AP database. Can you give an example of retrieving the values not stored in AP?